k8s/services/imagepullsecrets/helm-0.3.cue

```diff
@@ -0,0 +1,82 @@
+package imagepullsecrets
+
+import (
+	"guku.io/devx/k8s"
+	"k8s.io/api/core/v1"
+)
+
+#KubeVersion: [=~"^0\\.3\\."]: minor: >=21
+#Values: [=~"^0\\.3\\."]: {
+	replicas: uint | *1
+	istio: {
+		revision: string | *""
+	}
+	podAnnotations:     k8s.#Annotations | *{}
+	podSecurityContext: v1.#PodSecurityContext | *{
```

Comment:
since you set the schema the default values can be concrete 3ady

for example `true` is fine, you don't have to add `bool | *true`